modules/gdscript/gdscript_parser.cpp

```diff
@@ -5497,29 +5512,14 @@ void GDScriptParser::_determine_inheritance(ClassNode *p_class, bool p_recursive
 				}
 				p = nullptr;
 			} else {
```

Comment:
These diffs are hard to read, but they are the same code, just the old code had it's 'success' case nested in the loop.

Reply:
Diffs are easier to see when hiding whitespace.